packages/fluentui/docs/src/examples/components/Card/Usage/CardExampleFocusableChildrenGrid.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+  Flex,
+  Text,
+  Card,
+  Grid,
+  cardsContainerBehavior,
+  cardChildrenFocusableBehavior,
+  Button,
+  StarIcon,
+  DownloadIcon,
+  MoreIcon,
+} from '@fluentui/react-northstar';
+import { screenReaderContainerStyles } from '@fluentui/react-northstar';
+import * as React from 'react';
+import * as _ from 'lodash';
+
+type SampleCardProps = {
+  index?: number;
+};
+
+const SampleCard: React.FC<SampleCardProps> = ({ index, ...unhandledProps }) => {
+  return (
+    <Card accessibility={cardChildrenFocusableBehavior} aria-roledescription="user card" {...unhandledProps}>
+      <Card.Header>
+        <Text content={`User #${index}`} weight="bold" />
+      </Card.Header>
+      <Card.Footer>
+        <Flex space="between">
+          <Button content="Action" />
+          <Flex>
+            <Button icon={<StarIcon />} iconOnly text title="Favourite" />
+            <Button icon={<DownloadIcon />} iconOnly text title="Download" />
+            <Button icon={<MoreIcon />} iconOnly text title="More" />
+          </Flex>
+        </Flex>
+      </Card.Footer>
+    </Card>
+  );
+};
+
+const CardExampleFocusableGrid = () => {
+  const navigationMessageId = 'navigationMessage';
+  const defaultMessage = 'Press arrow keys to navigate between cards.';
+  const [navigationMessage, setNavigationMessage] = React.useState(defaultMessage);
+  const timeout = React.useRef<ReturnType<typeof setTimeout>>();
+  const cardsNumber = 18;
+  const handleGridFocus = () => {
+    if (navigationMessage !== null) {
+      timeout.current = setTimeout(() => {
+        setNavigationMessage(null);
+      }, 3000);
```

Comment:
is 3 second enough for us, because when I tested it long time ago for VoiceOver it was not enough and was not narrating the instruction message. 
The NVDA and JAWS works fine with the 3 sec, tested before while in this branch :)

Reply:
I've tested it in VoiceOver right now and with default setup 3 seconds was enough for it